ONSdigital/response-operations-ui
# Frontstage maintenance banner

# Motivation and Context

Creating a page that allows the team to easily set the maintenance banner on frontstage.

Previously this was achieved by manually updating the redis database

# What has changed



There is a new route /admin/banner which renders a new admin screen. In this screen you should be able to set the banner, update the banner and remove the banner.

# How to test?



1. Deploy to kubernetes
1. Port forward to both frontstage and response ops
1. Browser to /admin/banner on the response ops application
1. Set the banner
1. Browser to the sign in page on frontstage and check the banner is present
1. Update the banner
1. Check the banner is updated
1. Delete the banner
1. Check the banner is removed.

# Links




# Screenshots (if appropriate):
